### ISSUE TITLE
Backend crash (but recovers) when TABLE_FILE is directory

### ISSUE THREAD

**Issue by steffenslavetinsky** [Renumics/spotlight#65]
**Describe the bug**
Spotlight backend crashed (but recovers) when TABLE_FILE is pointing to a directory instead of a file.
For me this only happens when i launch it in dev mode.

**To Reproduce**

1. Run `TABLE_FILE=data/tables/ make dev`

**Expected behavior**
I would expect the backend not to fail;)

**Desktop (please complete the following information):**

-   OS: [e.g. iOS]
-   Browser Chrome 
-   Browser Version 113.0.5672.126 (Official Build) (arm64)
-   Spotlight Version 1.0.0

**Additional context**

The error that appears on the backend:

```bash
File ".../spotlight/renumics/spotlight_plugins/core/api/layout.py", line 42, in reset_layout
    dataset_uid = request.app.data_source.get_uid()
                  │       └ <property object at 0x165582930>
                  └ <starlette.requests.Request object at 0x1660284c0>

AttributeError: 'NoneType' object has no attribute 'get_uid'
```

**Comment by druzsan** [comment]
Should be fixed by #74

**Comment by druzsan** [comment]
Should be fixed by #74
